src/main/kotlin/org/opensearch/commons/notifications/model/SesAccount.kt

```diff
@@ -0,0 +1,124 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.commons.notifications.model
+
+import org.opensearch.common.Strings
+import org.opensearch.common.io.stream.StreamInput
+import org.opensearch.common.io.stream.StreamOutput
+import org.opensearch.common.io.stream.Writeable
+import org.opensearch.common.xcontent.ToXContent
+import org.opensearch.common.xcontent.XContentBuilder
+import org.opensearch.common.xcontent.XContentParser
+import org.opensearch.common.xcontent.XContentParserUtils
+import org.opensearch.commons.notifications.NotificationConstants.FROM_ADDRESS_TAG
+import org.opensearch.commons.notifications.NotificationConstants.REGION_TAG
+import org.opensearch.commons.notifications.NotificationConstants.ROLE_ARN_TAG
+import org.opensearch.commons.utils.logger
+import org.opensearch.commons.utils.validateEmail
+import org.opensearch.commons.utils.validateIamRoleArn
+import java.io.IOException
+
+/**
+ * Data class representing SES account channel.
+ */
+data class SesAccount(
+    val awsRegion: String,
+    val roleArn: String?,
+    val fromAddress: String
```

Comment:
I remember previously the decision was to only have a text box in UI for SES sender, did this change? are we adding a new config type for SesAccount, and another one for SES?

Reply:
SES configuration needs to be taken from UI instead of taking from yaml file hence additional fields. In UI, for the email configuration, customer can either choose SmtpAccount or SesAccount for sender profile.

Reply:
i see. also what's the format to specify `awsRegion`? is there a list of valid strings?